### ISSUE TITLE
[Windows] 'tee' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by lxhmavis** [Azure/vscode-kubernetes-tools#364]
Build: 4688 

Repro steps:

1. Right click the local minikube cluster in kubernetes clusters explorer
2. Select Open dashboard 

The result in the terminal as below:
![image](https://user-images.githubusercontent.com/41565966/45356067-e006f480-b5f4-11e8-9760-86ce26f26cd0.png)

**Comment by itowlson** [comment]
cc @bnookala

**Comment by bdelsaz** [comment]
I have the same problem. Specs:
> kubernetes extension 1.0.4
> vs code 1.38.1 
> Windows_NT x64 10.0.17763
> Chrome 69.0.3497.128
> Node.js 10.11.0

**Comment by casp4** [comment]
Same issue in mid 2021

**Comment by bnookala** [comment]
hiya folks, wanted to provide a quick update as i've been doing a bit of research on this issue

the root of this problem stems from windows running a command in cmd.exe rather than powershell (which is able to understand Tee) _however_, digging a bit deeper, i've also found:

- the kubernetes dashboard is no longer deployed in a kubernetes cluster, and is considered an _optional_ component, so it may not exist (and does not exist on newly provisioned AKS or GKE clusters)
- providers, specifically AKS and GKE, now direct users to utilize other dashboards (built into the Azure Portal or Cloud Console, respectively)
- other providers (ie EKS), direct users to the dashboard docs to perform a manual setup.

in summary: we _may_ revisit the dashboard experience, details to follow!
